Database/Corrections/cataQuestFixes.lua

```diff
@@ -4531,6 +4531,9 @@ function CataQuestFixes.Load()
         [26540] = { -- Dangerous Compassion
             [questKeys.extraObjectives] = {{nil, Questie.ICON_TYPE_TALK, l10n("Go back to the ship"),0,{{"monster",43188}}}},
         },
+        [26546] = { -- Razorbeak Friends
+            [questKeys.extraObjectives] = {{nil, Questie.ICON_TYPE_INTERACT, l10n("Feed Trained Razorbeak"), 0, {{"monster",2657}}}},
```

Comment:
Hey @gewoonnico actually it's better to just change the NPC that is listed in the `objectives` to the correct one. That way players will also see the progress on the tooltip.

You should also override the icon used for it:

```suggestion
            [questKeys.objectives] = {{{2657,nil,Questie.ICON_TYPE_INTERACT}}},
```

The new translation can also be removed then, because with `nil` Questie uses the string from the game.